### PR TITLE
MAUI - Added Header and Footer Feature Content in .NET MAUI SfAutoComplete and SfComboBox Controls

### PR DESCRIPTION
**Pull request by prakashKannanSf3972**
## Description ##

Prepared UG for Header and Footer feature in .NET MAUI SfAutoComplete and SfComboBox Controls.

### CODE REVIEW

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+# Header and Footer in Maui AutoComplete (SfAutoComplete)
+
+## Header and Footer in Maui SfAutoComplete
+
+We can provide Header and Footer view in the suggestion list in [`SfAutoComplete`] by enabling [`ShowDropDownHeaderView`] and [`ShowDropDownFooterView`]. 
+
+## Header Content
+
+We can provide Header Content at the top of the AutoComplete's Suggestion box.[`DropDownHeaderView`] property is used to set the content of the header. The height of the Header in the [`SfAutoComplete`] can be adjusted by the property [`DropDownHeaderViewHeight`].
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Header-and-Footer.md:18]: Use dropdown instead of suggestion list, address in whole PR

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+            this.Content = stackLayout;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Events
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Header-and-Footer.md:137]: Why did you add it in this section, please remove it.

File: MAUI/Autocomplete/Header-and-Footer.md (hunk 1 above)
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Header-and-Footer.md:18]: Need to enable the ShowHeaderView property to show the header view, please notify it.

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+            SearchLabel.Text = "Search for " + e.Value;
+        }
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+## Footer Content
+
+We can provide Footer Content at the bottom of the AutoComplete's Suggestion box. [`DropDownFooterView`] property is used to set the content of the footer.The height of the Header in the [`SfAutoComplete`] can be adjusted by the property [`DropDownFooterViewHeight`].
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Header-and-Footer.md:156]: Need to enable the ShowDropDownFooterView property to show the footer view, please notify it.

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+             xmlns:local="clr-namespace:AutocompleteSample"             
+             x:Class="AutocompleteSample.MainPage">
+
+    <StackLayout VerticalOptions="Start" 
+                 HorizontalOptions="Start" 
+                 Padding="30">
+        <autocomplete:SfAutoComplete HeightRequest="40"
+                                     ShowDropDownFooterView ="True"
+                                     DropDownFooterViewHeight="50">
+            <autocomplete:SfAutoComplete.AutoCompleteSource>
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Header-and-Footer.md:177]: Check the property name is ItemsSource or AutoCompleteSource

File: MAUI/ComboBox/Header-and-Footer.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+You can provide content for header at the top of the ComboBox's Suggestion box. The [`DropDownHeaderView`] property is used to set the content of the header. The height of the header in the [`SfComboBox`] can be adjusted using the [`DropDownHeaderViewHeight`] property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <StackLayout VerticalOptions="Start" HorizontalOptions="Start" Padding="30">
+        <combobox:SfComboBox HeightRequest="40" x:Name="comboBox" IsEditableMode="true" AllowFiltering="true">
+            <combobox:SfComboBox.DataSource>
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/ComboBox/Header-and-Footer.md:23]: Check the property name is ItemsSource or DataSource

File: maui-toc.html [hunk 6]
```diff
@@ -166,6 +166,7 @@
             <li><a href="/maui/Autocomplete/Getting-Started">Getting Started</a></li>
             <li><a href="/maui/Autocomplete/Migration">Migrate from Xamarin.Forms</a></li>
             <li><a href="/maui/Autocomplete/Selection">Selection</a></li>
+            <li><a href="/maui/AutoComplete/header-and-footer">Header and Footer</a></li>
```
Comment (naveenkumar-sanjeevirayan) [inline maui-toc.html:169]: Check the file name, Heade-and-Footer

File: maui-toc.html [hunk 7]
```diff
@@ -449,6 +450,7 @@
             <li><a href="/maui/ComboBox/Migration">Migrate from Xamarin.Forms</a></li>
             <li><a href="/maui/ComboBox/Editing">Editing</a></li>
             <li><a href="/maui/ComboBox/Selection">Selection</a></li>
+            <li><a href="/maui/ComboBox/header-and-footer">Header and Footer</a></li>
```
Comment (naveenkumar-sanjeevirayan) [inline maui-toc.html:453]: Check the file name, Heade-and-Footer

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title: Header and Footer in MAUI AutoComplete control | Syncfusion
+description: Learn about Header and Footer support in Syncfusion MAUI AutoComplete (SfAutoComplete) control and more details.
+platform: maui
+control: SfAutoComplete
+documentation: ug
+---
+
+# Header and Footer in .NET MAUI AutoComplete (SfAutoComplete)
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Header-and-Footer.md:10]: # Header and Footer support in .NET MAUI Autocomplete (SfAutoComplete)

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+
+## Header and Footer in .NET MAUI SfAutoComplete
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Header-and-Footer.md:12]: why 2 title used?

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header Image](Images/HeaderFooter/headertemplate.png)
+
+## Footer Content
+
+We can provide Footer Content at the bottom of the AutoComplete's dropdown. [`DropDownFooterView`] property is used to set the content of the footer.The height of the Header in the [`SfAutoComplete`] can be adjusted by the property [`DropDownFooterViewHeight`].
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Header-and-Footer.md:116]: Used header in the content

File: maui-toc.html (hunk 6 above)
Comment (Choza-rajan) [inline maui-toc.html:169]: See the folder name is wrong used the AutoComplete -> Autocomplete

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 11]
```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title: Header and Footer in MAUI AutoComplete control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Header-and-Footer.md:3]: Autocomplete

File: MAUI/Autocomplete/Header-and-Footer.md [hunk 12]
```diff
@@ -0,0 +1,211 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header Image](Images/HeaderFooter/headertemplate.png)
+
+## Footer Content
+
+We can provide Footer Content at the bottom of the Autocomplete's dropdown. [`DropDownFooterView`] property is used to set the content of the footer.The height of the Header in the [`SfAutocomplete`] can be adjusted by the property [`DropDownFooterViewHeight`].
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Header-and-Footer.md:114]: The height of the Header in the [`SfAutocomplete`] can be adjusted by the property
---
Header -> wrong word

File: MAUI/ComboBox/Header-and-Footer.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: Header and Footer in MAUI ComboBox control | Syncfusion
+description: Learn here all about Header and Footer support in Syncfusion Maui ComboBox (SfComboBox) control and more.
+platform: maui
+control: SfComboBox
+documentation: ug
+---
+
+# Header and Footer in .NET MAUI ComboBox (SfComboBox)
```
Comment (Choza-rajan) [inline MAUI/ComboBox/Header-and-Footer.md:10]: Header and Footer support in .NET MAUI ComboBox (SfComboBox)

File: maui-toc.html [hunk 14]
```diff
@@ -166,6 +166,7 @@
             <li><a href="/maui/Autocomplete/Getting-Started">Getting Started</a></li>
             <li><a href="/maui/Autocomplete/Migration">Migrate from Xamarin.Forms</a></li>
             <li><a href="/maui/Autocomplete/Selection">Selection</a></li>
+            <li><a href="/maui/Autocomplete/header-and-footer">Header and Footer</a></li>
```
Comment (Choza-rajan) [inline maui-toc.html:169]: use Header-and-Footer

File: maui-toc.html (hunk 7 above)
Comment (Choza-rajan) [reply maui-toc.html:453]: use Header-and-Footer

File: MAUI/ComboBox/Header-and-Footer.md (hunk 13 above)
Comment (prakashKannanSf3972) [reply MAUI/ComboBox/Header-and-Footer.md]: Added.

File: MAUI/Autocomplete/Header-and-Footer.md (hunk 12 above)
Comment (prakashKannanSf3972) [reply MAUI/Autocomplete/Header-and-Footer.md]: Changed header to Footer.

File: maui-toc.html (hunk 14 above)
Comment (prakashKannanSf3972) [reply maui-toc.html]: Changed.

**Review by Choza-rajan — APPROVED** [review]
